x/inflation/v1/keeper/integration_test.go

```diff
@@ -262,55 +347,69 @@ var _ = Describe("Inflation", Ordered, func() {
 						BeforeEach(func() {
-							params := s.app.InflationKeeper.GetParams(s.ctx)
+							params := s.network.App.InflationKeeper.GetParams(s.network.GetContext())
 							params.EnableInflation = true
-							s.app.InflationKeeper.SetParams(s.ctx, params) //nolint:errcheck
-
-							epochInfo, _ := s.app.EpochsKeeper.GetEpochInfo(s.ctx, epochstypes.DayEpochID)
-							epochNumber := epochInfo.CurrentEpoch // 6
-
-							epochsPerPeriod := int64(1)
-							s.app.InflationKeeper.SetEpochsPerPeriod(s.ctx, epochsPerPeriod)
-							skipped := s.app.InflationKeeper.GetSkippedEpochs(s.ctx)
-							s.Require().Equal(epochNumber, epochsPerPeriod+int64(skipped))
-
-							provision = s.app.InflationKeeper.GetEpochMintProvision(s.ctx)
-
-							s.CommitAfter(time.Hour * 23) // commit before next full epoch
-							provisionAfter := s.app.InflationKeeper.GetEpochMintProvision(s.ctx)
-							s.Require().Equal(provisionAfter, provision)
-
-							s.CommitAfter(time.Hour * 2) // commit after next full epoch
+							err := integrationutils.UpdateInflationParams(
+								integrationutils.UpdateParamsInput{
+									Tf:      s.factory,
+									Network: s.network,
+									Pk:      s.keyring.GetPrivKey(0),
+									Params:  params,
+								},
+							)
+							Expect(err).ToNot(HaveOccurred(), "error while setting params")
+
+							skipped := s.network.App.InflationKeeper.GetSkippedEpochs(s.network.GetContext())
+							Expect(skipped > uint64(0)).To(BeTrue())
+
+							epochsPerPeriod := s.network.App.InflationKeeper.GetEpochsPerPeriod(s.network.GetContext())
+							provision = s.network.App.InflationKeeper.GetEpochMintProvision(s.network.GetContext())
+
+							// commit before next full epoch
+							Expect(s.network.NextBlockAfter(time.Hour * 23)).To(BeNil())
+							provisionAfter := s.network.App.InflationKeeper.GetEpochMintProvision(s.network.GetContext())
+							Expect(provisionAfter).To(Equal(provision))
+
+							// commit after next full epoch (next period)
+							for i := int64(0); i < epochsPerPeriod; i++ {
+								Expect(s.network.NextBlockAfter(time.Hour * 24)).To(BeNil())
+							}
+
+							epochInfo, _ := s.network.App.EpochsKeeper.GetEpochInfo(s.network.GetContext(), epochstypes.DayEpochID)
+							epochNumber := epochInfo.CurrentEpoch
+
+							Expect(epochNumber > epochsPerPeriod).To(BeTrue())
 						})
 
 						It("should recalculate the EpochMintProvision", func() {
-							provisionAfter := s.app.InflationKeeper.GetEpochMintProvision(s.ctx)
+							provisionAfter := s.network.App.InflationKeeper.GetEpochMintProvision(s.network.GetContext())
 							Expect(provisionAfter).ToNot(Equal(provision))
-							Expect(provisionAfter).To(Equal(math.LegacyMustNewDecFromStr("159375000000000000000000000").Quo(math.LegacyNewDec(inflationkeeper.ReductionFactor))))
+							Expect(provisionAfter).To(Equal(math.LegacyMustNewDecFromStr("436643835616438356164384").Quo(math.LegacyNewDec(inflationkeeper.ReductionFactor))))
```

Comment:
Why is this value changed?

Reply:
this type of values change because the setup changed